drivers/sdcard/sdcard.py

```diff
@@ -226,41 +223,41 @@ def count(self):
         return self.sectors
 
     def readblocks(self, block_num, buf):
-        nblocks, err = divmod(len(buf), 512)
```

Comment:
`divmod()` apparently does a heap allocation.

Reply:
Yes it allocates a tuple.

Reply:
Ah, of course. That explains it.